src/components/widgets/NumericInput.js

```diff
@@ -80,15 +80,23 @@ export default class NumericInput extends Component {
   }
 
   incrementValue(direction) {
-    const {defaultValue, min, step = 1} = this.props;
+    const {defaultValue, min, step = 1, stepmode = 'absolute'} = this.props;
     const {value} = this.state;
 
     let valueUpdate;
     if (isNumeric(value)) {
       if (direction === 'increase') {
-        valueUpdate = parseFloat(value) + step;
+        if (stepmode === 'absolute') {
```

Comment:
this one just screams "ternary expression" :)

Reply:
fair enough :)